Securonix fix (#19697)
demisto/content: Packs/Securonix/ReleaseNotes/1_2_0.md

#### Integrations
##### Securonix
- Update the incident name created in the **fetch-incidents** command.
 The incident name will be according to the *Threat Model* field, if it does not exist it will be according to the *Policy* field

>  and if neither of them exist it will be the incident ID with the violator ID.

Reviewer: ```suggestion
 If neither of them exist, the incident name will be the incident ID combined with the violator ID.
```

>  This update is due to a change in Securonix API and suggestions from Securonix support team.

Reviewer: ```suggestion
 This update is due to a change in the Securonix API and with suggestions from the Securonix support team.
```
